docs/setup/install/targz.asciidoc

```diff
@@ -33,12 +33,13 @@ The Linux archive for Kibana v{version} can be downloaded and installed as follo
 ["source","sh",subs="attributes"]
 --------------------------------------------
 wget https://artifacts.elastic.co/downloads/kibana/kibana-{version}-linux-x86_64.tar.gz
-shasum -a 512 kibana-{version}-linux-x86_64.tar.gz <1>
+wget https://artifacts.elastic.co/downloads/kibana/kibana-{version}-linux-x86_64.tar.gz.sha512
+shasum -a 512 -c kibana-{version}-linux-x86_64.tar.gz.sha512 <1>
```

Comment:
Looks good, but while verifying the `-c` flag I realized we could avoid the sha512 file "trash" with:

```
curl https://artifacts.elastic.co/downloads/kibana/kibana-{version}-darwin-x86_64.tar.gz.sha512 | shasum -a 512 -c -
```